$MirOS: ports/print/cups/patches/patch-systemv_cancel_c,v 1.2 2006/06/21 19:25:36 bsiegert Exp $
--- systemv/cancel.c.orig	Tue Sep  5 22:45:24 2006
+++ systemv/cancel.c	Wed Nov  8 18:01:54 2006
@@ -280,7 +280,7 @@ main(int  argc,				/* I - Number of comm
       }
       else
       {
-        sprintf(uri, "ipp://localhost/jobs/%d", job_id);
+        snprintf(uri, sizeof(uri), "ipp://localhost/jobs/%d", job_id);
 	ippAddString(request, IPP_TAG_OPERATION, IPP_TAG_URI, "job-uri", NULL,
 	             uri);
       }
